tools/oni2pcd.cpp

```diff
@@ -83,7 +83,7 @@ main (int argc, char **argv)
   }
 
   pcl::ONIGrabber* grabber = new pcl::ONIGrabber (argv[1], false, false);
-  std::function<void (const CloudConstPtr&) > f = boost::bind (&cloud_cb, _1);
+  std::function<void (const CloudConstPtr&) > f = [=] (const CloudConstPtr& cloud) { cloud_cb (cloud); };
```

Comment:
Don't need to capture here. But does not really matter :)

Reply:
Then I'll change it, don't merge yet. I wasn't sure the function could be trivially captured inside the closure.